lib/log.js

```diff
@@ -62,6 +62,14 @@ function sanitise(raw) {
     raw = sanitiseConfigVariable(raw, 'CR_AGENT_URL');
```

Comment:
question: I noticed that `CR_AGENT_URL` is sanitized here while we didn't add anything for `GIT_CLIENT_URL`. Is this OK?

Reply:
Good question, we probably should. @shinin-light what do you think?

Reply:
Good call, I didn't think URLs would be eligible for sanitization, but if one is let's be consistent and sanitize them all.

Reply:
Ok, will add this.